Makefile.toml

```diff
@@ -395,8 +395,28 @@ fi
 
 # TODO: fix clippy lints in "sources" and check that too
 
+# For bash first-party source code
```

Comment:
Can we add this under a `tasks.check-shell` or something? Then we could potentially clean up our GitHub Action jobs to only trigger the Rust linting on `*.rs` file changes and ShellCheck on shell file changes.

Then of course also add this new task to `tasks.check` so the full suite can be easily run.